test/unit/expressions.spec.ts

```diff
@@ -177,10 +177,10 @@ test.each(["a>>b", "a>>=b"])("Unsupported bitop 5.3 (%p)", input => {
 
 test.each([
     { input: "1+1", lua: "1 + 1;" },
-    { input: "-1+1", lua: "(-1) + 1;" },
-    { input: "1*30+4", lua: "(1 * 30) + 4;" },
```

Comment:
Hmmm I'm not too sure about this removal, can we be sure operator prescedence is always the same in TS as it is in lua? What about adding a testcase with input `10 - (4 + 5)`.